packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6198,3 +6232,27 @@ The application CSR object defines a certificate signing request for a signature
 | csr              | Base64 encoded CSR in DER format                             | String                                                                      | TRUE     | TRUE   | TRUE     |           |           |            |
 | id               | unique identifier for the CSR                                | String                                                                      | FALSE    | TRUE   | TRUE     |           |           |            |
 | kty              | cryptographic algorithm family for the CSR's keypair         | String                                                                      | FALSE    | FALSE  | TRUE     |           |           |            |
+
+### ACS Endpoint Object
+
+The ACS endpoint containing the ACS URI and index of the URI.
```

Comment:
The ACS endpoint that contains the ACS URI and the index of the URI.

Reply:
done